HW2_Voskoboinikov/calculator.py

```diff
@@ -0,0 +1,35 @@
+def addition(a, b):
+	return(a + b)
+
+def subtraction(a, b):
+	return(a - b)
+
+def multiplication(a, b):
+	return(a * b)
+
+def division(a, b):
+	return(a / b)
+
+def calc_parser(calc_string):
+    calc_list = calc_string.split(' ')
+    calc_list[0] = float(calc_list[0])
+    calc_list[2] = float(calc_list[2])
+    return calc_list
+
+calculator_func_dict = {'-': subtraction,
+                        '*': multiplication,
+			'+': addition,
+			'/': division}
+
+calc_string = input('Enter your expression: ')
+
+while calc_string != 'exit':
+    command = calc_parser(calc_string)[1]
```

Comment:
Можно один раз вызвать функцию calc_parser() и потом разом записать в переменные элементы в список, а не трижды ее вызывать. Так работает, но выглядит неоптимально.
Например:
```suggestion
    numb1, command, numb2 = calc_parser(calc_string)
```